profiles/views_test.py

```diff
@@ -64,6 +65,27 @@ def setUp(self):
             )
         self.url2 = reverse('profile-detail', kwargs={'user': username})
 
+        # Monkey patch get_serializer_class so its return value can be asserted. This is reverted in tearDown
```

Comment:
can you help me understand the intention here? the comment indicates that we want to assert the return value for `get_serializer_class`, but i don't see where we're actually making that assertion